themes/classic/templates/catalog/_partials/product-cover-thumbnails.tpl

```diff
@@ -0,0 +1,20 @@
+<div class="images-container">
+  {block name='product_cover'}
+    <div class="product-cover _relative">
+      <img class="js-product-cover" src="{$product.cover.large.url}" alt="{$product.cover.legend}" title="{$product.cover.legend}" width="452" itemprop="image" />
```

Comment:
it is not related to this PR but I think we should use image size from configuration instead of hardcoded value